Integrations/integration-ServiceNow.yml

```diff
@@ -529,6 +528,33 @@ script:
         return dicRes;
     }
 
+
+    var SNGetGroups = function(name){
+        var parameters={};
+
+        var ticket_type = "sys_user_group";
+        var path = "table/"+ticket_type + encodeToURLQuery(parameters);
+        res = sendRequest('GET',path).obj;
+
+        var md = '## ServiceNow groups\n';
+        md += 'Id|Name|Description|Email|Active|Manager|Parent\n';
+        md += '---|---|---|---|---|---|---\n';
+
+        for (var i = 0; i < res.result.length; i++) {
+            md += res.result[i].sys_id + '|';
+            md += res.result[i].name + '|';
+            md += stringEscapeMD(res.result[i].description, true) + '|';
+            md += res.result[i].email + '|';
+            md += res.result[i].active + '|';
+            if (res.result[i].manager) {
+                md += res.result[i].manager.value + '|';
+            }
+            md += res.result[i].parent + '\n';
+        }
+
+        return {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': res, "HumanReadable": md, "EntryContext": {}};
```

Comment:
put in context something like 
ServiceNowGroups [{
   GroupID: id
   GroupName: name
},
...]

Reply:
Added